templates/plugin/battery.conf.erb

```diff
@@ -0,0 +1,11 @@
+<Plugin "battery">
```

Comment:
for new templates we agreed on epp only: https://voxpupuli.org/docs/#reviewing-a-module-pr
Can you please convert this to epp?

Reply:
done